iphone/cli/commands/_build.js

```diff
@@ -3409,6 +3408,13 @@ iOSBuilder.prototype.createXcodeProject = function createXcodeProject(next) {
 								extFrameworkReference = key.split('_')[0];
 								xobjs.PBXFileReference[extFrameworkReference] = extObjs.PBXFileReference[extFrameworkReference];
 								xobjs.PBXFileReference[extFrameworkReference + '_comment'] = child.comment;
+
+								const currentFileRef = Object.assign({}, xobjs.PBXFileReference[extFrameworkReference]);
+
+								if (!currentFileRef.path.includes('extensions/') && currentFileRef.sourceTree === '"<group>"') {
+									xobjs.PBXFileReference[extFrameworkReference].path = `${ext.relPath}/${currentFileRef.path}`;
+									xobjs.PBXFileReference[extFrameworkReference].name = currentFileRef.path;
+								}
```

Comment:
Before, the build failed because the frameworks embedded into the targets would only have the relative path to the build directory, e.g. `MyTarget/MyFramework.framework` instead of the relative Titanium project path.